cli/test/smokehouse/test-definitions/dobetterweb.js

```diff
@@ -355,9 +355,9 @@ const expectations = {
           items: {
             0: {
               displayedAspectRatio: /^120 x 15/,
-              url: 'http://localhost:10200/dobetterweb/lighthouse-480x318.jpg?iar1',
+              url: 'http://localhost:10200/dobetterweb/lighthouse-1024x680.jpg?iar1',
             },
-            length: 1,
+            length: 2,
```

Comment:
What's the new one, should we add an expectation for it now?

Reply:
it's unrelated, just so happened that the other audit being tested here got caught up. Don't wanna add it here...cuz it is named specific to its audit... :/